File: app/src/main/java/danb/speedrunbrowser/holders/ProgressSpinnerViewHolder.java
package danb.speedrunbrowser.holders;

import android.content.Context;

import androidx.recyclerview.widget.RecyclerView;
import danb.speedrunbrowser.views.ProgressSpinnerView;

public class ProgressSpinnerViewHolder extends RecyclerView.ViewHolder {
    public ProgressSpinnerViewHolder(Context ctx) {
        super(new ProgressSpinnerView(ctx, null));
    }
}

File: app/src/main/java/danb/speedrunbrowser/holders/ProgressSpinnerViewHolder.kt
package danb.speedrunbrowser.holders

import android.content.Context
import android.view.LayoutInflater

import com.yayandroid.parallaxrecyclerview.ParallaxViewHolder
import danb.speedrunbrowser.R

class ProgressSpinnerViewHolder(ctx: Context) :
        ParallaxViewHolder((ctx.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater)
                .inflate(R.layout.fragment_progress, null)) {

    override fun getParallaxImageId(): Int {
        return R.id.parallax
    }
}
